main.go

```diff
@@ -99,6 +101,8 @@ func main() {
 	flag.BoolVar(&enableLogging, "enable-logging", true, "enable/disable logging for the whole installation")
 	flag.StringVar(&loggingAgent, "logging-agent", common.LoggingAgentAlloy, fmt.Sprintf("select logging agent to use (%s or %s)", common.LoggingAgentPromtail, common.LoggingAgentAlloy))
 	flag.StringVar(&eventsLogger, "events-logger", common.EventsLoggerAlloy, fmt.Sprintf("select events logger to use (%s or %s)", common.EventsLoggerAlloy, common.EventsLoggerGrafanaAgent))
+	flag.Var(&includeEventsInNamespaces, "include-events-in-namespaces", "List of namespaces to collect events in by default on workload clusters")
+	flag.Var(&excludeEventsFromNamespaces, "exclude-events-from-namespaces", "List of namespaces to filter out collected events from workload clusters")
```

Comment:
Maybe this naming and descriptions would look more consistent?

```suggestion
	flag.Var(&includeEventsFromNamespaces, "include-events-in-namespaces", "List of namespaces to collect events from on workload clusters (if empty, collect from all namespaces)")
	flag.Var(&excludeEventsFromNamespaces, "exclude-events-from-namespaces", "List of namespaces to exclude events from on workload clusters")
```

Reply:
Sure that works :) i'll do the change tomorrow in all places :)